packages/cactus-plugin-ledger-connector-iroha/src/main/typescript/transaction.ts

```diff
@@ -0,0 +1,491 @@
+import { Logger } from "@hyperledger/cactus-common";
+import {
+  LoggerProvider,
+  LogLevelDesc,
+  Http405NotAllowedError,
+} from "@hyperledger/cactus-common";
+import {
+  IrohaCommand,
+  IrohaQuery,
+  RunTransactionRequestV1,
+  RunTransactionResponse,
+} from "./generated/openapi/typescript-axios";
+
+import { RuntimeError } from "run-time-error";
+import * as grpc from "grpc";
+
+import {
+  GrantablePermission,
+  GrantablePermissionMap,
+} from "iroha-helpers-ts/lib/proto/primitive_pb";
+
+import { CommandService_v1Client as CommandService } from "iroha-helpers-ts/lib/proto/endpoint_grpc_pb";
+import { QueryService_v1Client as QueryService } from "iroha-helpers-ts/lib/proto/endpoint_grpc_pb";
+
+import commands from "iroha-helpers-ts/lib/commands/index";
+import queries from "iroha-helpers-ts/lib/queries";
+
+export class Transaction {
```

Comment:
@StepniowskiP Could this be called `IrohaTransactionWrapper` instead? The `Iroha` prefix will help with distinguishing it in the logs from every other ledger's transaction logs and the `Wrapper` will hint at the fact that it's not the class/type that ships with the Iroha libraries that we use.

Reply:
Sure, renamed to IrohaTransactionWrapper